datasetinsights/estimators/faster_rcnn.py

```diff
@@ -377,7 +377,7 @@ def log_metric_val(self, label_mappings, epoch):
                 [result_per_label for result_per_label in result.values()]
             )
             logger.info(f"metric {metric_name} has mean result: {mean_result}")
-            self.writer.add_scalar(f"val/m{metric_name}", mean_result, epoch)
+            self.writer.add_scalar(f"val/{metric_name}", mean_result, epoch)
```

Comment:
Do we want to change the name for `kfp_metrics` as well? Can we instead change metric name instead of adding `m` prefix?